src/test/scala/rx/lang/scala/ObservableTest.scala

```diff
@@ -428,4 +428,15 @@ class ObservableTests extends JUnitSuite {
     assertEquals(expectedMap3, m3.toBlocking.single)
   }
 
+  @Ignore // Don't run it. Only test if it can be compiled
+  def testErrorSignature() {
+    val o: Observable[Int] = Observable.error(new RuntimeException("Oops")).map(v => v)
+    println(o.toBlocking.single)
+
+    val x = 1
+    println(x + (Observable.error(new RuntimeException("Oops")).toBlocking.single: Int))
```

Comment:
@samuelgruetter Just revisited this one and found `error[T]` also needs to specify `: Int`. Now I cannot find a case that `error[T]` is better than `error[Nothing]`. So how about changing to `error[Nothing]`?